futures-util/src/lib.rs

```diff
@@ -0,0 +1,30 @@
+//! Combinators and utilities for working with futures-rs `Futures`s, `Stream`s, and `Sink`s.
+
+#![no_std]
+#![deny(missing_docs, missing_debug_implementations)]
+#![doc(html_root_url = "https://docs.rs/futures/0.1")]
+
+#[macro_use]
+extern crate futures_core;
+extern crate futures_executor;
+extern crate futures_sink;
+
+use futures_core::{Async, Future, IntoFuture, Poll, Stream};
+use futures_core::task;
+use futures_sink::{AsyncSink, Sink, StartSend};
+
+#[macro_use]
+#[cfg(feature = "std")]
+extern crate std;
+
+#[cfg(feature = "std")]
+pub mod lock;
+
+pub mod future;
+pub use future::FutureExt;
```

Comment:
Hm... we may wanna chat on IRC briefly about preludes/exports maybe to sort this out, I'm not so sure my prelude-per-crate idea will pan out

Reply:
Discussed on IRC-- decision was to keep individual preludes for crates and not to reexport items from dependencies.